testing_stratgey/tester.c

```diff
@@ -1,7 +1,7 @@
 #include "ICU.h"
 #include <stdio.h>
 #include <stdlib.h>
```

Comment:
You should provide a detailed description of all test cases;
Test Case ID:
Test Case Description:
Test Data:
Expected Results:
Actual Results:
Pass/Fail:

Reply:
Also, did you implement Stubs/Mocks for the ICU?